### ISSUE TITLE
Contractor and Separator from variables and Operation

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/IntervalConstraintProgramming.jl#131]
We need `Contractor(vars, ex::Operation)` and similarly for `Separator`.

cc @yashcodes

**Comment by dpsanders** [comment]
We should also remove `::Function` from the currently defined method `Contractor(vars, f::Function)` so that it can be anything that behaves like a `Function`.

**Comment by yashvardhan747** [comment]
`Separator(vars, expr::Operation )` is already there.

**Comment by yashvardhan747** [comment]
According to issue#117 I think we want something like 
```
julia>para = @polyvar x y
julia> p(x,y)  = x+y < 1  
julia> Separator( para, p )

 ```

**Comment by dpsanders** [comment]
Fixed by #133
